streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -4093,36 +4093,44 @@ object ZStream extends ZStreamPlatformSpecificConstructors {
     repeatEffectWith(UIO.succeed(a), schedule)
 
   /**
-   * Accesses the specified service in the environment of the effect.
+   * Accesses the specified service in the environment of the stream.
    */
   def service[A: Tag]: ZStream[Has[A], Nothing, A] =
     ZStream.access(_.get[A])
 
   /**
-   * Accesses the specified services in the environment of the effect.
+   * Accesses the specified services in the environment of the stream.
```

Comment:
This is a longer conversation but we need to delete all these arity versions and require users expand multiple service dependencies in a for comprehension.

Reply:
Yes.